mw-webapp/src/component/image/Image.tsx

```diff
@@ -25,21 +26,41 @@ interface ImageProps {
    * Data attribute for cypress testing
    */
   dataCy?: string;
+
+  /**
+   * Should image open?
+   */
+  isShouldOpen?: boolean;
 }
 
 /**
  * Сomponent for displaying images
  */
 export const Image = (props: ImageProps) => {
+
+  const [isOpen, setIsOpen] = useState(false);
+
   const className = clsx(styles.image, props.className);
 
+  /**
+   * Function full screen image
+   */
+  const onClickHandler = (value: boolean) => {
+    setIsOpen(value);
+  };
+
   return (
-    <img
-      src={props.src}
-      alt={props.alt}
-      className={className}
-      data-cy={props.dataCy}
-    />
+    <div className={isOpen ? styles.imageWrapperFullScreen : ""}>
+      <img
+        tabIndex={0}
+        src={props.src}
+        alt={props.alt}
+        className={className}
+        data-cy={props.dataCy}
+        onClick={props.isShouldOpen ? () => onClickHandler(true) : undefined}
+        onBlur={() => onClickHandler(false)}
```

Comment:
Maybe better to use Modal and render big image after click by trigger. 

@tritonJS826 What do you think?

If not, then need add possibility to close image not only by onBlur - it can be not clear for user

Reply:
I can add a transparent button in the corner with X

Reply:
@Ekaterina1994 added close button